src/lxc/criu.c

```diff
@@ -407,7 +407,7 @@ static int exec_criu(struct cgroup_ops *cgroup_ops, struct lxc_conf *conf,
 			return log_error_errno(-EIO, EIO, "Failed to freezer cgroup entry");
 
 		if (!opts->user->disable_skip_in_flight &&
-				strcmp(opts->criu_version, CRIU_IN_FLIGHT_SUPPORT) >= 0)
+		    strcmp(opts->criu_version, CRIU_IN_FLIGHT_SUPPORT) >= 0)
```

Comment:
Not converting that one?

Reply:
Ah no, because we want to check for greater than not just 0. :)

Reply:
Oh right, missed that.